README.md

```diff
@@ -43,6 +43,51 @@ projects into Fedora operating system.
 * Provide feedback on upstream pull requests.
   * Run RPM builds in COPR and integrate the results into Github.
 
+## Requirements for upstream projects
+
+If your project would like to use `packit`, there are some requirements which
+needs to be full field.
+
+* place in your upstream project directory `.packit.json`
```

Comment:
[Again](https://github.com/user-cont/colin/pull/222#issuecomment-463590666)? Seriously? ;-)

Reply:
What, JSON or YAML? But I don't know If we decide to place it in upstream or downstream.